package.json

```diff
@@ -69,80 +68,77 @@
-        "css-loader": "1.0.0",
-        "enzyme": "3.10.0",
-        "enzyme-adapter-react-16": "1.14.0",
-        "enzyme-to-json": "3.3.5",
-        "eslint": "6.8.0",
+        "core-js": "2.6.11",
+        "coveralls": "^3.0.11",
+        "css-loader": "^3.4.0",
+        "enzyme": "^3.11.0",
+        "enzyme-adapter-react-16": "^1.15.2",
+        "enzyme-to-json": "^3.4.4",
+        "eslint": "^6.8.0",
         "eslint-config-airbnb-typescript": "^7.2.0",
         "eslint-config-prettier": "^6.0.0",
         "eslint-loader": "3.0.3",
-        "eslint-plugin-import": "^2.20.1",
+        "eslint-plugin-import": "^2.20.2",
         "eslint-plugin-jsx-a11y": "^6.2.3",
         "eslint-plugin-prettier": "^3.1.2",
         "eslint-plugin-react": "^7.19.0",
         "eslint-plugin-react-hooks": "^2.0.1",
-        "file-loader": "4.2.0",
-        "html-webpack-plugin": "3.2.0",
-        "husky": "1.2.0",
-        "jest": "24.8.0",
-        "jest-css-modules-transform": "2.4.0",
-        "mini-css-extract-plugin": "0.4.3",
-        "node-sass": "4.12.0",
+        "file-loader": "^6.0.0",
+        "husky": "^4.2.3",
+        "jest": "^24.9.0",
+        "jest-css-modules-transform": "^4.0.0",
+        "mini-css-extract-plugin": "^0.9.0",
+        "node-sass": "^4.13.1",
         "npm-run-all": "4.1.5",
-        "optimize-css-assets-webpack-plugin": "5.0.1",
-        "postcss-import": "12.0.0",
-        "postcss-loader": "3.0.0",
-        "precss": "3.1.2",
+        "optimize-css-assets-webpack-plugin": "^5.0.3",
+        "postcss-import": "^12.0.1",
+        "postcss-loader": "^3.0.0",
         "prettier": "^2.0.2",
         "react": "16.8.6",
         "react-docgen-typescript-loader": "3.4.0",
         "react-dom": "16.8.6",
         "react-icons": "3.7.0",
         "react-router-dom": "4.3.1",
         "react-transition-group": "4.3.0",
-        "regenerator-runtime": "0.13.3",
-        "sass-loader": "7.1.0",
-        "standard-version": "^6.0.1",
-        "style-loader": "0.23.0",
+        "regenerator-runtime": "^0.13.5",
+        "sass-loader": "^8.0.0",
+        "standard-version": "^7.1.0",
+        "style-loader": "^1.1.0",
         "stylelint": "^13.2.1",
         "stylelint-config-recommended": "^3.0.0",
         "stylelint-config-recommended-scss": "^4.2.0",
         "stylelint-order": "^4.0.0",
         "stylelint-scss": "^3.16.0",
-        "stylelint-webpack-plugin": "1.2.3",
-        "terser-webpack-plugin": "1.2.3",
-        "ts-jest": "24.1.0",
-        "ts-loader": "6.2.1",
+        "stylelint-webpack-plugin": "^1.2.3",
+        "ts-jest": "^24.3.0",
+        "ts-loader": "^6.2.2",
         "typescript": "^3.8.3",
-        "webpack": "4.28.1",
-        "webpack-bundle-analyzer": "3.3.2",
-        "webpack-cli": "3.2.1",
-        "webpack-dev-server": "3.1.14"
+        "webpack": "^4.42.1",
+        "webpack-bundle-analyzer": "^3.6.1",
+        "webpack-cli": "^3.3.11",
+        "webpack-dev-server": "^3.10.3"
```

Comment:
What do you think about updating other libraries? 
<img width="485" alt="Screenshot 2020-04-01 at 11 00 40" src="https://user-images.githubusercontent.com/19436304/78119109-34a4d700-7408-11ea-84b7-4fefdd8670ae.png">

Reply:
next subtask